From 9807374501c71def499998024ee038426e1a8dc0 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 13 Sep 2020 13:22:15 +0300
Subject: [PATCH] [SW-17444] preboot: goya2_arc: increase hash nodes

Since goya2 has zephyr images along with uboot image to be verified,
the max hash nodes currently set to 10 won't suffice. This patch
increases max hash nodes to 20.

Change-Id: I97b791635b6809d5690ad039d866bd39859bc61f
---

diff --git a/preboot/fit.c b/preboot/fit.c
index dcac2df..647d5dc 100644
--- a/preboot/fit.c
+++ b/preboot/fit.c
@@ -35,7 +35,7 @@
 
 #define FIT_MAX_PATH		200
 #define MAX_FDT_REGIONS		10
-#define IMAGE_MAX_HASHED_NODES	10
+#define IMAGE_MAX_HASHED_NODES	20
 
 struct fdt_region fdt_regions[MAX_FDT_REGIONS];
 struct image_region region[MAX_FDT_REGIONS];
